manoseimas/mps_v2/migrations/0001_initial.py

```diff
@@ -2,6 +2,8 @@
 from __future__ import unicode_literals
 
 from django.db import models, migrations
+import autoslug.fields
+import manoseimas.mps_v2.models
```

Comment:
It's not exactly PEP-8 compliant but renaming everything would be a hassle we will probably have to repeat if we swap `mps` with `mps_v2` at a later point.